ayon_server/settings/enum.py

```diff
@@ -130,6 +132,71 @@ async def anatomy_presets_enum():
     return result
 
 
+def anatomy_template_items_enum(
+    project_name: str | None = None,
+    category: Literal["work", "publish", "hero", "delivery", "others", "staging"] | None = None
+):
+    """ Provides values of template names from Anatomy as dropdown.
+
+    Wrapper for actual function as Settings require callable.
+
+    If no category is required it must be used as:
+        `enum_resolver=anatomy_template_items_enum()`
+    Args:
+        project_name: str
```

Comment:
Based on this https://github.com/ynput/ayon-backend/pull/558/files#r2028918590 the argument can be removed from docstring.